pandas/tests/groupby/test_function.py

```diff
@@ -1507,14 +1507,35 @@ def test_quantile_missing_group_values_no_segfaults():
         grp.quantile()
 
 
-def test_quantile_missing_group_values_correct_results():
+@pytest.mark.parametrize(
+    "key, value",
+    [
+        (["a"] * 4 + ["b"] * 4, np.array([1.0, 2.0, 3.0, np.nan] * 2)),
```

Comment:
Any way to make these examples more succinct? Somewhat of a personal nit, but smaller tests and ideally constructing from literal values makes things easier for the reader

Reply:
Updated to make these a bit smaller / more readable